tests/test-empress.js

```diff
@@ -1,125 +1,127 @@
-require(["jquery", "BPTree", "Empress", "BiomTable", "util", "chroma"], function($, BPTree, Empress, BiomTable, util, chroma) {
-    $(document).ready(function() {
+require([
+    "jquery",
+    "BPTree",
+    "Empress",
+    "BiomTable",
+    "util",
+    "chroma",
+], function ($, BPTree, Empress, BiomTable, util, chroma) {
+    $(document).ready(function () {
         // Setup test variables
         // Note: This is ran for each test() so tests can modify bpArray without
         // effecting other test
-        module('Empress' , {
-            setup: function() {
+        module("Empress", {
+            setup: function () {
                 // tree comes from the following newick string
                 // ((1,(2,3)4)5,6)7;
                 var tree = new BPTree(
                     new Uint8Array([1, 1, 1, 0, 1, 1, 0, 1, 0, 0, 0, 1, 0, 0]),
-                    null, null, null);
+                    null,
+                    null,
+                    null
+                );
                 var layoutToCoordSuffix = {
-                    "Rectangular": "r",
-                    "Circular": "c2",
-                    "Unrooted": "2"
+                    Rectangular: "r",
+                    Circular: "c2",
+                    Unrooted: "2",
                 };
 
                 var nameToKeys = {
-                    "root": [7],
-                    "EmpressNode6": [6],
-                    "internal": [5, 4],
+                    root: [7],
+                    EmpressNode6: [6],
+                    internal: [5, 4],
                     "2": [2],
                     "3": [3],
-                    "1": [1]
+                    "1": [1],
                 };
                 // Note: the coordinates for each layout are "random". i.e.
                 // they will not make an actual tree. They were created to
                 // make testing easier.
                 var treeData = {
                     7: {
-                        "color":[1.0, 1.0, 1.0],
-                        "inSample": false,
-                        "xr": 13,
-                        "yr": 14,
-                        "xc2": 27,
-                        "yc2": 28,
-                        "x2": 41,
-                        "y2": 42,
-                        "name": "root",
-                        "inSample": true,
-                        "visible": true
+                        color: [1.0, 1.0, 1.0],
+                        xr: 13,
+                        yr: 14,
+                        xc2: 27,
+                        yc2: 28,
+                        x2: 41,
+                        y2: 42,
+                        name: "root",
+                        inSample: true,
```

Comment:
```suggestion
                        inSample: false,
```

This should actually be false. It looks like there was actually a mistake in the test.